easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.6-centos6-03Apr2019.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = "Tarball"
+
+name = 'LS-PrePost'
+version = '4.6'
+versionsuffix = '-centos6-03Apr2019'
```

Comment:
the date is not related to the centos version but to the the release date? then maybe replace this with:
```
release_date = '03Apr2019'
versionsuffix = '-centos6'
```
and also update the easyconfig name.

Reply:
Thanks, I'm not sure why I thought it was some build version suffix. It being a release date seems much more reasonable. I have done as you suggested.